pages/documentation/developer/api_guide/short-circuit.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: default
+---
+
+# Short circuit API
```

Comment:
Here the text is justified but the rendering looks a bit weird because of the long name `...ShortCircuitAnalysisProvider`. Is it possible to do something about it?
![image](https://user-images.githubusercontent.com/93923177/227490913-62f025c2-b38d-412b-ae26-4fb2799dc515.png)

Reply:
For all problems with the justified text: I changed a bit the text to improve the rendering. It's not a perfect solution though but I couldn't figure out something better.